src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -183,8 +187,51 @@ object DatabricksPlugin extends AutoPlugin {
     }
   }
 
+  private lazy val executeCommandImpl: Def.Initialize[Task[Seq[CommandStatus]]] = Def.task {
+    val client = dbcApiClient.value
+    val language = dbcExecutionLanguage.value
+    val onClusters = dbcClusters.value
+    val (allClusters, _) = dbcFetchClusters.value
+    val commandFile = dbcCommandFile.value
+    val commandStatuses = Seq.empty[CommandStatus]
+    client.foreachCluster(onClusters, allClusters) { confirmedCluster =>
+      var contextCorrect = false
+      val contextId = client.createContext(language, confirmedCluster)
+      while (!contextCorrect) {
+        val contextStatus = client.checkContext(contextId, confirmedCluster)
+        if (contextStatus.status == DBCContextRunning.status) {
+          contextCorrect = true
+        } else if (contextStatus.status == DBCContextError.status) {
+          client.destroyContext(contextId, confirmedCluster)
+          contextCorrect = true
+          Def.task(throw new RuntimeException(s"The Execution Context '${contextId.id} returned an error. Terminated context"))
```

Comment:
You don't need the `Def.task`'s  around Runtime Exceptions. I have those in other places as hacks. To force sequential execution.